crates/luwen-if/src/chip/blackhole.rs

```diff
@@ -344,6 +344,18 @@ impl Blackhole {
     pub fn spi_write(&self, mut addr: u32, value: &[u8]) -> Result<(), Box<dyn std::error::Error>> {
         let buffer = self.get_spi_buffer()?;
 
+        let (status, _, _) = match self.bh_arc_msg(0xC2, None, &[0, 0], None) {
+            Ok(status) => status,
+            Err(_) => {
+                println!("Warning, flash unlock message not recognized");
```

Comment:
Not sure if we want the print here or not? In tt-flash it shows up like this:

```
 Sub Stage FLASH Step 2: Blackhole[0] {p150a}
                        Writing new firmware... (this may take up to 1 minute)Warning, flash unlock message not recognized
Warning, flash lock message not recognized
                        Writing new firmware... SUCCESS
                        Firmware verification... SUCCESS
```

This isn't an issue here, but if tt-flash issues multiple `spi_write` calls to luwen (which it will do if the firmware bundle is a sparse hex file) we would probably get this message multiple times.

@sbansalTT or @kikkiakki, thoughts?

Reply:
I'll add the reason we need to catch this error at all is because we need this code to keep working on current versions of the FW that don't accept this message, and work on new releases that do.